@here/harp-mapview/lib/MapView.ts

```diff
@@ -506,6 +506,13 @@ export interface MapViewOptions {
      */
     enablePhasedLoading?: boolean;
 
+    /**
+     * Enable map repeat. If `false`, map will be repeated on X axis continuously.
```

Comment:
1. If true, the map will repeat
2. only for planar projections
3. instead of X axis I would reword it in longitudal direction (i.e. there is no X axis in geo space)
4. also mention where it will repeat/wrap around (i.e. data-border or antimeridian)